HealthcareApp/src/Healthcare.Infrastructure/Persistance/AppDbContext.cs

```diff
@@ -11,39 +12,50 @@ public class AppDbContext : IdentityDbContext<ApplicationUser>
 {
     public DbSet<Schedule> Schedules { get; set; }
     public DbSet<Appointment> Appointments { get; set; }
-    
+
+    public int? DatePart(string datePartArg, DateTime? date) => throw new Exception();
```

Comment:
???

Reply:
it was workaround when I could not optiomize query. when done, I just forgot to remove it